Week2/week2.txt

```diff
@@ -0,0 +1,35 @@
+1. create table stud*/
+ create table stud
+ (
+ id int primary key,
+ name text
+ );
+
+2. create table student_detail*/
+
+create table stud_detail
+(
+sid int,
+city text,
+mobile_no int,
+foreign key (sid) references stud(id)
+);
+
+---- insert queries of student -------
+
+ insert into stud values(1,'jay'),(2,'happy'),(3,'om'),(4,'sai'),(5,'karan'),insert into stud values(6,'vatsal'),(7,'meet'),(8,'harsh'),(9,'amit'),(10,'hiren');

```

Comment:
Why you can write insert query two time to insert value in same table?

Reply:
by mistake i done

Reply:
In most relational database systems, executing the same INSERT query twice for the same table will result in duplicate data being inserted. Each time you execute an INSERT query, a new row is added to the table with the specified data. If you execute the same INSERT query multiple times, it will insert the same data multiple times, creating duplicate entries in the table.